components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/internal/OAuth2ServiceComponentHolder.java

```diff
@@ -66,4 +71,32 @@ public static RegistryService getRegistryService() {
     public static void setRegistryService(RegistryService registryService) {
         OAuth2ServiceComponentHolder.registryService = registryService;
     }
+
```

Comment:
How about using How about moving this to OpenIDConnectServiceComponentHolder since claim specific tasks are connected more to OIDC rather than pure OAuth :wink:

Reply:
Now there is no internal package for OpenIdConnect. Then needs to create OpenIDConnectServiceComponentHolder.